src/network/networkHandler.ts

```diff
@@ -117,22 +117,20 @@ export function onData(d: OnDataArgs, overworld: Overworld) {
           }
           // End Report Differences to evaluate where client server player desyncs are ocurring
 
-
           // Host loads player data from client to syncronize the state
           Player.load(player, foundPlayerIndex, overworld.underworld, false);
         }
       }
       break;
-    case MESSAGE_TYPES.SET_GAME_MODE:
+    }
+    case MESSAGE_TYPES.SET_GAME_MODE: {
       const { gameMode } = payload;
       if (underworld.levelIndex <= 1) {
         underworld.gameMode = gameMode;
         // Must be called when difficulty (gameMode) changes to update summon spell stats
         Cards.refreshSummonCardDescriptions(underworld);
         recalculateGameDifficulty(underworld);
-        // Clear lastLevelCreated in order to allow it to regenerate the level without
-        // changing the levelIndex
-        underworld.lastLevelCreated = undefined;
```

Comment:
Why not clear lastLevelCreated anymore?

Reply:
4150257d25d3f93f26f27bcab1716e479e1b41f6 made a fix elsewhere that makes this redundant/no longer needed. The commit changes and description give more information 👍